x-pack/plugins/secops/server/lib/hosts/index.ts

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { HostsData } from '../../../common/graphql/types';
+import { FrameworkRequest } from '../framework';
+export * from './elasticsearch_adapter';
+import { HostsAdapter, HostsRequestOptions } from './types';
+
+export class Hosts {
+  private adapter: HostsAdapter;
+
+  constructor(adapter: HostsAdapter) {
+    this.adapter = adapter;
```

Comment:
Optional. I really like this pattern you have used before on the backend:

```ts
constructor(private readonly adapter: HostsAdapter) { }
```

But, again, optional. So much good work I wouldn't hold up the PR over somethings this simple. Just pointing it out.